kvmagent/kvmagent/plugins/tensorfusion/executor.py

```
@@ -0,0 +1,304 @@
+    def is_alive(worker):
+        # type: (Worker) -> bool
+        """Check if a worker process is still alive."""
+        try:
+            import psutil
+            p = psutil.Process(worker.pid)
+            return p.is_running() and p.status() != psutil.STATUS_ZOMBIE
+        except ImportError:
+            # Fallback: use kill(0)
+            try:
+                os.kill(worker.pid, 0)
+                return True
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)